lib/broadway.ex

```diff
@@ -950,7 +966,11 @@ defmodule Broadway do
     metadata = Map.new(Keyword.get(opts, :metadata, []))
 
     ref = make_ref()
-    ack = {Broadway.CallerAcknowledger, {self(), ref}, :ok}
+
+    ack_module = Keyword.get(opts, :acknowledger_module, Broadway.CallerAcknowledger)
```

Comment:
@jeffgrunewald @josevalim I have a feeling that acknowledger_module is a bit long name. Maybe `ack_module` and `ack_data` would be more appropriate in the end?